docs/example/manifests/access/traffic-access-v1-allow-bookthief.yaml

```diff
@@ -40,3 +40,62 @@ spec:
     pathRegex: ".*a-book.*new"
     methods:
     - GET
+---
+kind: TrafficTarget
+apiVersion: access.smi-spec.io/v1alpha3
+metadata:
+  name: bookstore-access-bookwarehouse
+  namespace: bookwarehouse
+spec:
+  destination:
+    kind: ServiceAccount
+    name: bookwarehouse
+    namespace: bookwarehouse
+  rules:
+  - kind: HTTPRouteGroup
+    name: bookwarehouse-service-routes
+    matches:
+    - restock-books
+  sources:
+  - kind: ServiceAccount
+    name: bookstore
+    namespace: bookstore
+---
+apiVersion: specs.smi-spec.io/v1alpha4
+kind: HTTPRouteGroup
+metadata:
+  name: bookwarehouse-service-routes
+  namespace: bookwarehouse
+spec:
+  matches:
+    - name: restock-books
+      methods:
+      - POST
+---
+kind: TrafficTarget
+apiVersion: access.smi-spec.io/v1alpha3
+metadata:
+  name: mysql
+  namespace: bookwarehouse
+spec:
+  destination:
+    kind: ServiceAccount
+    name: mysql
+    namespace: bookwarehouse
+  rules:
+  - kind: TCPRoute
+    name: tcp-route
+  sources:
+  - kind: ServiceAccount
+    name: bookwarehouse
+    namespace: bookwarehouse
+---
+apiVersion: specs.smi-spec.io/v1alpha4
+kind: TCPRoute
+metadata:
+  name: tcp-route
```

Comment:
Could you rename this from `tcp-route` to something more indicative of its use? Maybe `mysql`? Or possibly `mysql-tcp`?